client/me/sidebar/index.jsx

```diff
@@ -151,6 +153,11 @@ class MeSidebar extends Component {
 						/>
 					</SidebarMenu>
 				</SidebarRegion>
+				<CollapseSidebar
+					key="collapse"
```

Comment:
In this case, the `key` prop isn't necessary. It looks like you copied this pattern from the sidebar code in My Sites, but we only add [the special `key` prop](http://reactjs.org/docs/lists-and-keys.html) when we're rendering elements in a loop. (or sometimes when we're doing a fact re-render hack, but that's pretty uncommon).

React logs key usage errors to the console in dev mode.

Reply:
I don't think I've ever seen a lit rule for `key` before. But it does look like there's rules around the place: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-key.md

I'm not sure it's the sort of thing that linters could get right 100% of the time, so I'm not sure what I'd think about trying to rely on linting for it.